clientd3d/ownerdrw.c

```diff
@@ -330,6 +330,7 @@ void DrawOwnerListItem(const DRAWITEMSTRUCT *lpdis, Bool selected, Bool combo)
    AREA area;
    COLORREF crColorText;
    HBRUSH hColorBg;
+   enum special_type special_type_value = SPECIAL_TYPE_NONE;
```

Comment:
No "enum" here.

Reply:
changed `enum` to `typedef enum` in `proto.h` as suggested and adjusted related function parameters